docs/index.rst

```diff
@@ -0,0 +1,69 @@
+Finetune documentation
+====================================
+
+.. module:: finetune
+
+Finetune is a python library designed to make finetuning pre-trained language models
+for custom natural language processing tasks a breeze.
+
+It ships with pre-trained model weights
+from `"Improving Language Understanding by Generative Pre-Training" <https://s3-us-west-2.amazonaws.com/openai-assets/research-covers/language-unsupervised/language_understanding_paper.pdf>`_
+and builds off the `OpenAI/finetune-language-model repository <https://github.com/openai/finetune-transformer-lm>`_.
+
+Source code for finetune is available `on github <https://github.com/IndicoDataSolutions/finetune-language-model>`_.
+
+
+Installation
+============
+Finetune can be installed directly from PyPI by using `pip`
+
+.. code-block:: bash
+
+    pip install finetune
+
+
+or installed directly from source:
+
+.. code-block:: bash
+
+    git clone https://github.com/IndicoDataSolutions/finetune
+    cd finetune
+    python3 setup.py develop
+
+You can optionally run the provided test suite to ensure installation completed successfully.
+
+.. code-block:: bash
+
+    nosetests
+
+
+Finetune Quickstart Guide
+=========================
+
+Finetuning the base language model is as easy as calling :meth:`LanguageModelClassifier.fit`:
+
+.. code-block:: python3
+
+    model = LanguageModelClassifier()   # load base model
+    model.fit(trainX, trainY)           # finetune base model on custom data
+    predictions = model.predict(testX)  # predict on unseen examples
+    model.save(path)                    # serialize the model to disk
+
+Easily reload saved models from disk by using :meth:`LanguageModelClassifier.load`:
+
+.. code-block:: python3
+
+    model = LanguageModelClassifier.load(path)
+    predictions = model.predict(testX)
```

Comment:
it would be helpful to see the output of this - I spent hours yesterday faffing around with this ticket - to find the results were inadequate for what I need. 
https://github.com/Franck-Dernoncourt/NeuroNER/issues/115

if the model needs more collaboration/ effort - that's fine - garbage in / garbage out. 
I want to see the results from an example. 
eg
nn.predict('From 1987 until 2000 NIPS was held in Denver, United States. Since then, the conference was held in Vancouver, Canada (2001–2010), Granada, Spain (2011), and Lake Tahoe, United States (2012–2013)')

Reply:
Reasonable suggestion, I'll add that in.